deps/rabbitmq_auth_backend_oauth2/src/uaa_jwt.erl

```diff
@@ -67,21 +84,33 @@ decode_and_verify(Token) ->
 
 -spec get_jwk(binary()) -> {ok, map()} | {error, term()}.
 get_jwk(KeyId) ->
-    case signing_keys() of
-      undefined -> {error, signing_keys_not_configured};
```

Comment:
Wasn't really sure about this case.. it would have meant duplicating the JWKS code updating code "just" to preserve the error message for a very similar case.